django/forms/fields.py

```diff
@@ -290,8 +290,13 @@ def widget_attrs(self, widget):
 class FloatField(IntegerField):
     default_error_messages = {
         'invalid': _('Enter a number.'),
+        'step_size': _('Ensure number is of correct step size.'),
     }
 
+    def __init__(self, *, max_value=None, min_value=None, step_size='any', **kwargs):
```

Comment:
set default `step_size=None` instead of `"any"` and only render that attribute if it's `not None`.